src/lib.rs

```diff
@@ -248,6 +243,18 @@ pub fn resolve(args: ClapperArgs) -> Result<(), Box<dyn Error>> {
     Ok(())
 }
 
+fn collect_arguement_from_the_terminal() -> String {
```

Comment:
There is already an existing pattern of getting command line inputs, don't reinvent the wheel.